eras/shelley/impl/src/Cardano/Ledger/Shelley/LedgerState.hs

```diff
@@ -69,6 +73,10 @@ module Cardano.Ledger.Shelley.LedgerState
 
     -- * Epoch boundary
     stakeDistr,
+    incrementalStakeDistr,
+    updateStakeDistribution,
+    resolveIncrementalPtrs,
```

Comment:
Is there a reason to export this? Seems like it's only used in `incrementalStakeDistr`, and since it has a fairly heavy use restriction on it, would be better to keep it internal.
```suggestion
```

Reply:
done, I also realized it does not have to return an IncrementalStake, it needs only return the resolved map. So that removes its 'heavy' restriction, and since it is not used elsewhere I removed it from the exports.